tests/Fixtures/e2e/Test_Framework_Options_Config/tests/SourceClassTest.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Test_Framework_Options_Config\Test;
+
+use Test_Framework_Options_Config\SourceClass;
+use PHPUnit\Framework\TestCase;
+
+class SourceClassTest extends TestCase
+{
+    public function test_hello()
+    {
+        if (
+            !in_array('--strict-global-state', $GLOBALS['argv'], true)
+        ) {
+            var_dump($GLOBALS['argv']);
```

Comment:
I would suggest using `$this->fail('Failure to pass framework option)'`here and for `initialTestsPhpOptions` test